dot/core/runtime_test.go

```diff
@@ -46,7 +51,7 @@ func TestRetrieveAuthorityData(t *testing.T) {
 		rt: rt,
 	}
 
-	auths, err := s.retrieveAuthorityData()
+	auths, err := s.grandpaAuthorities()
```

Comment:
`retrieveAuthorityData` called `grandpaAuthorities`, I removed `retrieveAuthorityData` for now